build/Dockerfile

```diff
@@ -15,6 +15,15 @@ RUN MACH=$(uname -m); if [[ ${MACH} == "aarch64" ]]; then ARCH=arm64; \
     else echo "Unsupported arch: ${MACH}"; ARCH=${MACH}; fi; \
     wget -O /usr/local/bin/kubectl "https://dl.k8s.io/release/$(wget -qO - https://dl.k8s.io/release/stable.txt)/bin/linux/${ARCH}/kubectl" && \
     chmod +x /usr/local/bin/kubectl
+# Download the latest helm in the appropriate architecture. Currently handles aarch64 (arm64) and x86_64 (amd64).
+RUN MACH=$(uname -m); if [[ ${MACH} == "aarch64" ]]; then ARCH=arm64; \
+    elif [[ ${MACH} == "x86_64" ]]; then ARCH=amd64; \
+    elif [[ ${MACH} == "armv7l" ]]; then ARCH=arm; \
+    else echo "Unsupported arch: ${MACH}"; ARCH=${MACH}; fi; \
+    wget https://get.helm.sh/helm-v3.10.3-linux-${ARCH}.tar.gz && \
+    tar -zxvf helm-v3.10.3-linux-${ARCH}.tar.gz && \
+    mv linux-${ARCH}/helm /usr/local/bin/helm && \
+    chmod +x /usr/local/bin/helm
```

Comment:
I believe if you delete the `.tar.gz ` in the same `RUN` statement, it will actually be deleted - same image layer.

Or you can pipe wget into tar, something like:
```
RUN mkdir -p /usr/src/things \
    && curl -SL https://example.com/big.tar.xz \
    | tar -xJC /usr/src/things \
    && make -C /usr/src/things all
```

[docker docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy)

Reply:
Piping is sth that I would avoid here, but removing the `tar.gz` make sens 👍